<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shv::visu::logview::DataViewWidget</class>
 <widget class="QWidget" name="shv::visu::logview::DataViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>25</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lblDataViews">
     <property name="text">
      <string>Data view:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbDataView">
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="pbShowChannelFilterDialog">
     <property name="toolTip">
      <string>Channels filter</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../images/images.qrc">
       <normaloff>:/shv/visu/images/settings.svg</normaloff>:/shv/visu/images/settings.svg</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
